packages/react/src/components/atoms/buttons/ButtonTag/index.js

```diff
@@ -25,8 +33,10 @@ ButtonTag.propTypes = {
   type: PropTypes.string.isRequired,
   /** The label text of the sort button */
   value: PropTypes.string.isRequired,
-  /** The label text of the sort button */
-  text: PropTypes.string.isRequired,
+  /** The content of the button */
+  text: PropTypes.node.isRequired,
+  /** The visually hidden description of the button */
+  description: PropTypes.node,
```

Comment:
why is the description a node? Can't it just be a string, since it's for screen reader only?

Reply:
Value of this prop is rendered into the content. It could be anything that is a valid React node. It doesn't look like we should additionally restrict this prop. What if developers want to add additional markup for screen readers, e.g. a few HTML tags with `aria-*` attributes?